sys/fuchsia/fidlgen/main.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2018 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+var zirconLibs = []string{
+	"fuchsia-process",
+	"fuchsia-io",
+}
+
+func main() {
+	targetArch := os.ExpandEnv("${TARGETARCH}")
+	arch := ""
+	switch targetArch {
+	case "amd64":
+		arch = "x64"
```

Comment:
We already have this in 'sys/targets/targets.go'. So you can use `targets.Get(...).KernelHeaderArch`. But probably worth checking that Get does not return nil.

Reply:
Done with nil check.